sdk/textanalytics/ai-text-analytics/samples/javascript/authenticateWithAad.js

```diff
@@ -2,27 +2,29 @@
 // Licensed under the MIT License.
 
 /**
- * uses Azure Active Directory (RBAC) to authenticate to the Cognitive Services endpoint
+ * Demonstrates how to use Azure Active Directory (RBAC) to authenticate
+ * a TextAnalyticsClient
  */
 
 const { TextAnalyticsClient } = require("@azure/ai-text-analytics");
 const { DefaultAzureCredential } = require("@azure/identity");
 
 // Load the .env file if it exists
-require("dotenv").config();
+const dotenv = require("dotenv");
```

Comment:
Can you add a sample in this file that also includes how to authenticate with an api key. I know Python is doing this, and it'd be nice to be able to know how to authenticate both ways by looking at this sample